lib/GameConstants.h

```diff
@@ -53,6 +53,9 @@ namespace GameConstants
 	constexpr int DEFAULT_SCHOOLS = 4;
 	constexpr int CRE_LEVELS = 10; // number of creature experience levels
 
+	constexpr size_t HERO_BACKPACK_WINDOW_SLOT_COLUMNS = 8;
+	constexpr size_t HERO_BACKPACK_WINDOW_SLOT_LINES = 8;
+

```

Comment:
Should probably be inside hero backpack class. This is not global constant but local constant